frontera/contrib/scrapy/middlewares/seeds/__init__.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
Porting guides mostly recommend making this change for pragmatic reasons - [1 ](https://docs.python.org/3/howto/pyporting.html#prevent-compatibility-regressions),   [2 ](https://wiki.python.org/moin/PortingPythonToPy3k),   [3 ](http://python-future.org/compatible_idioms.html#imports-relative-to-a-package).
If you think it is adding needless noise, most of these statements can be removed without any affect(just a final confirmation from you).